runtime/caml/gc_ctrl.h

```diff
@@ -27,6 +27,21 @@ void caml_init_gc ();
 value caml_gc_stat(value);
 value caml_gc_major(value);
 
+
+#define caml_stat_top_heap_wsz caml_top_heap_words(Caml_state->shared_heap)
+#define caml_stat_compactions 0
+#define caml_stat_heap_wsz caml_heap_size(Caml_state->shared_heap)
```

Comment:
I think there is a units error here. Shouldn't this be:
`Wsize_bsize(caml_heap_size(Caml_state->shared_heap))`